### DISCUSSION TITLE
Allow adding tags in wp-config

### DISCUSSION THREAD

**Discussion by jonatcorus** [stayallive/wp-sentry#112] [Ideas]
It would be useful to be able to add tags in the wp-config. This could be used by automated deploy systems for adding metadata to a specific deployment. I'm not sure of the best way this would be done. It could be through a define such as `define('WP_SENTRY_TAGS', '{"tag1": "val1", "tag2": "val2"}');` or a define with a PHP array  `define('WP_SENTRY_TAGS', ["tag1" => "val1", "tag2" => "val2"]);` It would then get picked up along with the default tags here https://github.com/stayallive/wp-sentry/blob/697d72d61b079ba332e366c2bb7e366661cb4b65/src/class-wp-sentry-php-tracker.php#L181-L185

**Comment by stayallive** [comment]
I think this would be a nice addition, I'm thinking that maybe there is more I want to expose through a `define` since on PHP 7 we can now `define` arrays and some default tags would be a nice addition.

Thanks for requesting this!
